presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -890,4 +890,77 @@ CTE Materialization Properties
 
     The name of the catalog to be used for Common Table Expressions (CTE) and which provides custom partitioning for Common Table Expression (CTE) materialization.
     This setting specifies which catalog should be used for CTE materialization and for determining how to partition the materialization of CTEs in queries.
-    This can also be specified on a per-query basis using the ``cte_partitioning_provider_catalog`` session property.
\ No newline at end of file
+    This can also be specified on a per-query basis using the ``cte_partitioning_provider_catalog`` session property.
+
+
+Logging Properties
+------------------
+
+``log.path``
+^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Default value:** ``var/log/server.log``
+
+    The path to the log file used by Presto. The path is relative to the data
+    directory, configured by the launcher script.
+
+``log.max-history``
+^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``30``
+
+    The maximum number of general application log files to use, before log
+    rotation replaces old content.
+
+``log.max-size``
+^^^^^^^^^^^^^^^^
+
+    * **Type:** ``data size``
+    * **Default value:** ``100MB``
+
+    The maximum file size for the general application log file.
+
+``http-server.log.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``true``
+
+    Flag to enable or disable logging for the HTTP server.
+
+``http-server.log.compression.enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``true``
+
+    Flag to enable or disable compression of the log files of the HTTP server.
+
+``http-server.log.path``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Default value:** ``var/log/http-request.log``
+
+    The path to the log file used by the HTTP server. The path is relative to
+    the data directory, configured by the launcher script as detailed in
+    :ref:`running_presto`.
+
+``http-server.log.max-history``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``15``
+
+    The maximum number of log files for the HTTP server to use, before
```

Comment:
`http-server.log.max-history` property is a little more complex than it is named. I have played around with this in the past and it doesn't directly correlate to the number of files.

So, Max History controls the number of archive log periods to keep. 
Log Period is determined from the FileNamePattern. In our case, it's a daily rollover (https://github.com/prestodb/airlift/blob/a5f238a6ba3426264fad89422fabaf5ef3ae0f7d/http-server/src/main/java/com/facebook/airlift/http/server/DelimitedRequestLog.java#L94).
So, 1 log period in our case will be 1 day. If we set `http-server.log.max-history` to 5, that would mean it will keep logs for 5 log periods (i.e. days). 

More information on TimeBasedRollingPolicy(which airlift is using) from logback: https://logback.qos.ch/manual/appenders.html#TimeBasedRollingPolicy